rancher/service_prune.go

```diff
@@ -17,6 +20,7 @@ func pruneServices(context *Context, project *project.Project) error {
 			if err != nil {
 				return err
 			}
+			logrus.Infof("Deleting service %s", service.Name)
```

Comment:
Is it better to print the deleted services in one line? something like `Deleted service: s1, s2, s3`

Reply:
Since we start services one by one, I don't think there's any issue with showing that we delete them one by one.